features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/ApiCreate.jsx

```diff
@@ -41,12 +42,22 @@ const ApiCreate = () => {
                                                 <div className='ch-info'>
                                                     <div className='ch-info-front ch-img-1'>
                                                         <i className='fw fw-document fw-4x' />
-                                                        <span>I Have an Existing API</span>
+                                                        <span>
+                                                            <FormattedMessage
+                                                                id='i.have.an.existing.api'
+                                                                defaultMessage='I Have an Existing API'
+                                                            />
+                                                        </span>
                                                     </div>
                                                     <div className='ch-info-back'>
                                                         <p className='unselectable'>
-                                                            Use an existing API endpoint or the API Swagger definition
-                                                            to create an API
+                                                            <FormattedMessage
+                                                                id='use.an.existing.api.endpoint.or.the.api.
+                                                                swagger.definition.to.create.an.api'
+                                                                defaultMessage='Use an existing API
```

Comment:
Don't we have a way of getting the locale-en value as the default value in here? If we can do that, we don't need to define the same message in two places.

Reply:
Yes I think @Rajith90  has already done that in here [#diff-e5aa33738269df52f6d813d4cd2830abR66](https://github.com/wso2/carbon-apimgt/pull/5583/files#diff-e5aa33738269df52f6d813d4cd2830abR66) , But the problem is if we don't provide a defaultMessage i18n library will Render the message id as the text with capitalized words

Reply:
@pubudu538 It read the value from locale-en.json file first, if it is missing then it gets the default value we specified. If the default value is missing then it will add the key value we provided in uppercase letters. So as a practice its better to provide the default value as well.